iceoryx_posh/include/iceoryx_posh/internal/roudi/roudi_multi_process.hpp

```diff
@@ -41,17 +41,33 @@ using namespace iox::units::duration_literals;
 class RouDiMultiProcess
 {
   public:
+
+    // indicate whether the message queue thread will start directly or deferred
+    // this is important for derived classes which may need to initialize their members before the thread starts
+    enum class MQThreadMode {
```

Comment:
why not call it like in the comment above?
```
enum class MqThreadStart
{
    DIRECT,
    DEFERRED
}
```
This would make the intention more obvious when reading, e.g. `MqThreadStart::DIRECT` ... maibe `IMMEDIATE` would also work

Reply:
Yes, would probably be more explicit and make the intention obvious.

Reply:
its renamed